types/move-concurrently/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for move-concurrently 1.0
+// Project: https://www.npmjs.com/package/move-concurrently
+// Definitions by: Melvin Groenhoff <https://github.com/mgroenhoff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/**
+ * Recursively moves from to to and resolves its promise when finished. If to already exists then the promise will be rejected with an EEXIST error.
+ */
+declare function move<T extends PromiseLike<void> = Promise<void>>(from: string, to: string, opts?: move.Options): T;
```

Comment:
I think i tried something like that but couldn't get it to work quickly. This seems to works. Thanks!